modules/containers_proto/proto/hephaestus/containers/proto/bit_flag.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package heph.containers.proto;
+
+message BitFlag {
+  uint64 value = 1;
+}
```

Comment:
missing empty line at the end of the file. You can add a setting to your VS Code to automatically add an empty line